### DISCUSSION TITLE
Detecting BB center coordinate for kv images

### DISCUSSION THREAD

**Discussion by tsengt01** [jrkerns/pylinac#422] [Q&A]
Hello,

I would like to know if it is possible to use any of the module in pylinac to retract the BB center coordinate for planar kv (Varian OBI) images just like what can be done for EPID images with the Winston-Lutz test?

Thank you so much in advance.

Amber

**Comment by jrkerns** [comment]
"retract the BB center"? If you mean "extract", as in, find the BB center as part of a Winston-Lutz test, yes. See this section: https://pylinac.readthedocs.io/en/latest/winston_lutz.html#kv-analysis-imaging-only-iso-evaluation
